### PR TITLE
fix(modules-sdk): create workflow returning step transformer

### PR DESCRIPTION
**Pull request by carlos-r-l-rodrigues**
Fixes: https://github.com/medusajs/medusa/issues/6084

### CODE REVIEW

File: packages/workflows-sdk/src/helper/workflow-export.ts [hunk 1]
```diff
@@ -131,31 +131,38 @@ export const exportWorkflow = <TData = unknown, TResult = unknown>(
...
+                ? res.output
+                : res
+            })
+          } else {
+            const res = transaction.getContext().invoke?.[resFrom]
+            result =
+              res?.__type === OrchestrationUtils.SymbolWorkflowWorkflowData
+                ? res.output
+                : res
+          }
```
Comment (adrien2p) [inline packages/workflows-sdk/src/helper/workflow-export.ts]:
```suggestion
          const resFromArray = Array.isArray(resFrom) ? resFrom : [resFrom]
          result = resFromArray.map((from) => {
              const res = transaction.getContext().invoke?.[from]
              return res?.__type ===
                OrchestrationUtils.SymbolWorkflowWorkflowData
                ? res.output
                : res
            })
          result =   Array.isArray(resFrom) ? result : result[0]
```

**Review by adrien2p — APPROVED** [review]
LGTM, jus one suggestion wdyt?

**Review by adrien2p — APPROVED** [review]
LGTM 💪

**Comment by carlos-r-l-rodrigues** [comment]
@olivermrbl can I merge this one?

**Comment by olivermrbl** [comment]
yessir!

**Pull request merged**